### ISSUE TITLE
Collapse not working

### ISSUE THREAD

**Issue by zhryn** [bestguy/sveltestrap#56]
I'm trying to use a Navbar component. I have an issue with the collapse component. The hidden area won't show when I click the NavbarToggler component but the variable isOpen value has changed.

```
let isOpen = false;
const toggleNavbar = () => (isOpen = !isOpen);

<Navbar dark fixed="top" expand="lg" id="mainNav">
  <div class="container">
    <NavbarBrand href="/">
      <img src="/assets/img/navbar-logo.png" alt="" />
    </NavbarBrand>
    <NavbarToggler on:click={toggleNavbar}>
      Menu
      <i class="fa fa-bars" />
    </NavbarToggler>
    <Collapse isOpen={isOpen} navbar>
      <Nav class="ml-auto text-uppercase" navbar>
        <NavItem>
          <NavLink href="#page-top">Home</NavLink>
        </NavItem>
        <NavItem>
          <NavLink href="#page-top">About Us</NavLink>
        </NavItem>
      </Nav>
    </Collapse>
  </div>
</Navbar>
```

**Comment by bestguy** [comment]
Hi @zhryn looks like Collapse does not work on mobile size, I think there is some bad logic here: https://github.com/bestguy/sveltestrap/blob/master/src/Collapse.svelte#L31

I'm not sure why this conditional is there but will look to correct this weekend.

**Comment by bestguy** [comment]
Hi @zhryn sorry GitHub closed this issue automatically when I merged PR.

This should be corrected in sveltestrap 3.2.2 (https://github.com/bestguy/sveltestrap/releases/tag/v3.2.2) , can you please try and confirm it corrects your issue?
You can reopen if not.

**Comment by bestguy** [comment]
Hi @zhryn, this should be corrected now, please reopen if you see issue with version 3.2.2 or later.
